aidial_adapter_openai/dalle3.py

```diff
@@ -126,7 +127,7 @@ async def chat_completion(
             type="invalid_request_error",
         )
 
-    api_url = upstream_endpoint + "?api-version=2023-12-01-preview"
+    api_url = upstream_endpoint + f"?api-version={api_version}"
```

Comment:
This would generally work, but the better way to add a query parameter would be by using functions from urllib.parse:
Something like this:
```
api_url = urlparse(upstream_endpoint)._replace(query=urlencode({"api-version" : api_version})).geturl()
```
This would handle the cases like upstream_endpoint already ending with "?" character.

Reply:
Since upstream_endpoint comes from the dial core configuration, I think we can expect that this url has been validated and does not contain extra characters.